modules/nw-scaling-ingress-controller.adoc

```diff
@@ -52,8 +57,19 @@ $ oc get -n openshift-ingress-operator ingresscontrollers/default -o jsonpath='{
 ----
 3
 ----
-
-[NOTE]
++
+[TIP]
 ====
-Scaling is not an immediate action, as it takes time to create the desired number of replicas.
+You can alternatively apply the following YAML to scale an Ingress Controller:
+[source,yaml]
+----
+apiVersion: v1
+kind: IngressController
+metadata:
+  name:default
+  namespace:openshift-ingress-operator
+spec:
+  replicas: 3
+    name: custom-certs-default
```

Comment:
Need the API group and spaces, and `name: custom-certs-default` looks like leftover copypasta:
```suggestion
apiVersion: operator.openshift.io/v1
kind: IngressController
metadata:
  name: default
  namespace: openshift-ingress-operator
spec:
  replicas: 3
```
By the way, it is also possible to use `oc scale`:
```shell
oc scale -n openshift-ingress-operator ingresscontroller/default --replicas=3
```